campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How I came to Rust
+---
+
+# How I came to Rust
+
+The #rustreach program asked people to write down how they came to Rust and I'm happy to do that. Entering a community or a space is always a moment of orientation process and I hope I can give some insight into that.
```

Comment:
Hi! As a newbie, I'm not sure what the #rustreach program is... Perhaps some other people might come across the post who are new to it too? Perhaps you could link to a URL for it, and/or add a few words to summarise it? :)

Reply:
good point, for background here's the campaign issue: https://github.com/rust-community/content-o-tron/issues/38 

I'll create a separate issue to discuss if the "editorial assistance" tasks should have all the information linked (like this campaign issue) so editors can start editing.